EV_Monitor/demo_queries.py

```diff
@@ -0,0 +1,108 @@
+"""
+EV Monitor Demo Queries
+
+This script demonstrates various queries you can make to the EV Neighborhood Monitor.
+Run this after starting the main ev_monitor_app.py to see example interactions.
+"""
+
+import time
+import os
+from strands import Agent
+from strands.tools.mcp.mcp_client import MCPClient
+from mcp.client.streamable_http import streamablehttp_client
+from strands.models.anthropic import AnthropicModel
+
+# Configure the model
+model = AnthropicModel(
+    client_args={
+        "api_key": os.getenv("ANTHROPIC_API_KEY"),
+    },
+    max_tokens=2048,
+    model_id="claude-3-5-sonnet-20241022",
+    params={
+        "temperature": 0.3,
+    }
+)
```

Comment:
can we use bedrock nova
```suggestion
    client_args={
        "api_key": os.getenv("ANTHROPIC_API_KEY"),
    },
    max_tokens=2048,
    model_id="amazon nova",
    params={
        "temperature": 0.1,
    }
)
```